### PR TITLE
Adds wings from potion of flight for fly persons in game.

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
So, i was kinda surprised that there are no fly wings in game, when their sprite was already made. So it adds wings from strange elixir (potion of flight) for fly persons. Not much to discuss, i guess.
## Why It's Good For The Game
Wings for flies.
## Changelog
:cl:
add: Wings for fly persons from flight potion
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
These are _functional_ wings which means that this change allows flies to ascend Z levels and ignore holes even while under gravity. I feel like that would be worth noting in the OP.

**Comment by Helg2** [comment]
> These are _functional_ wings which means that this change allows flies to ascend Z levels and ignore holes even while under gravity. I feel like that would be worth noting in the OP and probably is something to discuss. If you want ones more like moth wings they would need a different typepath.
Basically as any other wings, yes. They wont get it from just being a fly though. But i'm not sure about adding it as analog to moth wings.

**Comment by Jacquerel** [comment]
Ah I see, these are from drinking a flight potion while transformed into a fly? I misunderstood.

File: code/modules/surgery/organs/external/wings/functional_wings.dm [hunk 1]
```diff
@@ -196,3 +196,9 @@
 	name = "megamoth wings"
 	desc = "Don't get murderous."
 	sprite_accessory_override = /datum/sprite_accessory/wings/megamoth
+
+//fly wings for flies
```
Comment (Time-Green) [inline code/modules/surgery/organs/external/wings/functional_wings.dm:200]: Three slashes (///) for an autodoc comment. You should specifiy these are from the flight potion as well

**Review by Time-Green — CHANGES_REQUESTED** [review]

**Comment by Time-Green** [comment]
You need to update your PR description and changelog to what this actually does. This does not give flies wings, this gives them a unique wingpotion variant. The distinction is extreme

**Comment by Helg2** [comment]
> You need to update your PR description and changelog to what this actually does. This does not give flies wings, this gives them a unique wingpotion variant. The distinction is extreme

Didn't thought about that.

File: code/modules/surgery/organs/external/wings/functional_wings.dm (hunk 1 above)
Comment (Helg2) [reply code/modules/surgery/organs/external/wings/functional_wings.dm]: That'll do?

**Review by Time-Green — APPROVED** [review]

**Comment by ShizCalev** [comment]
Shouldn't this be considered balance instead?

**Comment by Helg2** [comment]
> Shouldn't this be considered balance instead?

Why?

**Comment by tralezab** [comment]
> Shouldn't this be considered balance instead?

Like, technically, but think about the edge case you even get these wings: You have to be a flyperson drinking a potion a flight, a rare drop from lavaland. And the balance is that instead of getting angel wings that make you holy, you get normal wings that make you look like a fly. TRAIT_HOLY is not really a biggie

**Pull request merged**